### ISSUE TITLE
Dataloader crashes when training FBCCNN on a CSVDataset

### ISSUE THREAD

**Issue by acodeodyssey** [torcheeg/torcheeg#65]
When training the model FBCCNN on a CSVDataset, I get the following error in my logs: 

    RuntimeError('Input type (double) and bias type (float) should be the same')

I create the dataset with:

    dataset = CSVFolderDataset(csv_path=csv_path, 
                      read_fn=self.default_read_fn,
                      io_path=self.io_path,
                      before_trial=None,
                      online_transform=transforms.Compose(
                       [transforms.BandPowerSpectralDensity(sampling_rate=1000),
                        transforms.ToGrid(self.location_dict)]),                  
                      label_transform=transforms.Compose([transforms.Select('label')]),
                      num_worker=16)

Location Dict:

      CHANNEL_LIST = ['FP1', 'FPZ', 'FP2', 'AF3', 'AF4', 'F11', 'F7', 
                                             'F5', 'F3', 'F1', 'FZ', 'F2', 'F4', 'F6', 'F8', 'F12', 'FT11',
                                             'FC5', 'FC3',  'FC1','FCZ', 'FC2', 'FC4', 'FC6', 'FT12',
                                             'T7', 'C5', 'C3', 'C1', 'CZ', 'C2', 'C4', 'C6', 'T8', 'TP7', 
                                             'CP5', 'CP3', 'CP1', 'CPZ','CP2', 'CP4', 'CP6', 'TP8',
                                             'M1', 'M2', 'P7', 'P5', 'P3', 'P1', 'PZ', 'P2', 'P4', 'P6', 
                                             'P8', 'PO7', 'PO3', 'POZ', 'PO4', 'PO8', 
                                   'O1', 'OZ', 'O2', 'CB1', 'CB2']

      LOCATION_LIST = [
                                     ['-', '-', '-', '-', 'FP1', 'FPZ', 'FP2', '-', '-', '-', '-'],
                                     ['-', '-', '-', '-', 'AF3', '-', 'AF4', '-', '-', '-', '-'],
                                     ['F11', 'F7', 'F5', 'F3', 'F1', 'FZ', 'F2', 'F4', 'F6', 'F8', 'F12'],
                                     ['FT11', '-', 'FC5', 'FC3', 'FC1', 'FCZ', 'FC2', 'FC4', 'FC6', '-', 'FT12'],
                                     ['-', 'T7', 'C5', 'C3', 'C1', 'CZ', 'C2', 'C4', 'C6', 'T8', '-'],
                                     ['-', 'TP7', 'CP5', 'CP3', 'CP1', 'CPZ', 'CP2', 'CP4', 'CP6', 'TP8','-'],
                                     ['M1', 'P7', 'P5', 'P3', 'P1', 'PZ', 'P2', 'P4', 'P6', 'P8', 'M2'],
                                     ['-', '-', 'PO7', '-', 'PO3', 'POZ', 'PO4', '-', 'PO8', '-', '-'],
                                     ['-', '-', '-', '-', 'O1', 'OZ', 'O2', '-', '-', '-', '-'],
                                     ['-', '-', '-', '-', 'CB1', '-', 'CB2', '-', '-', '-', '-']
    
      self.location_dict = torcheeg.datasets.constants.utils.format_channel_location_dict(

Training:

            model = FBCNet(num_classes=2,num_electrodes=self.electrodes,in_channels=4,
                           chunk_size=config.chunk_size,stride_factor=4)

            trainer = ClassifierTrainer(model=model,
                                      lr=config.learning_rate,
                                      weight_decay=1e-4,num_classes=2,accelerator='cuda') 

            train_loader = DataLoader(train_dataset,
                                    batch_size=config.batch_size,num_workers=16
                                    )

            val_loader = DataLoader(val_dataset,
                            batch_size=config.batch_size,num_workers=16,
                            shuffle=False)

            # Do 50 rounds of training
            trainer.fit(train_loader, val_loader, max_epochs=self.n_epochs,logger=wandb_logger)

Could you help me, figure out the cause of this problem?

**Comment by tczhangzhi** [comment]
You need to convert the signal to a `FloatTensor` (instead of double) if you are using a `CSVDataset` dataset.
```

        dataset = CSVFolderDataset(
            csv_path=csv_path,
            read_fn=self.default_read_fn,
            io_path=self.io_path,
            before_trial=None,
            online_transform=transforms.Compose([
                transforms.BandPowerSpectralDensity(sampling_rate=1000),
                transforms.ToGrid(self.location_dict),
                transforms.ToTensor() # see here
            ]),
            label_transform=transforms.Compose([transforms.Select('label')]),
            num_worker=16)
```

**Comment by acodeodyssey** [comment]
@tczhangzhi Thank you, now it works
